src/gui/optionsdialog.cpp

```diff
@@ -1414,6 +1416,7 @@ void OptionsDialog::loadWebUITabOptions()
     connect(m_ui->textWebUIPassword, &QLineEdit::textChanged, this, &ThisType::enableApplyButton);
     connect(m_ui->btnWebUIAPIKeyCopy, &QPushButton::clicked, this, &ThisType::onBtnWebUIAPIKeyCopy);
     connect(m_ui->btnWebUIAPIKeyRotate, &QPushButton::clicked, this, &ThisType::onBtnWebUIAPIKeyRotate);
+    connect(m_ui->btnWebUIAPIKeyDelete, &QPushButton::clicked, this, &ThisType::onBtnWebUIAPIKeyDelete);
```

Comment:
I just noticed a flaw in all these event handler names. They should be of the form `onBtnNameClicked`.